open-machine-learning-jupyter-book/deep-learning/GAN.md

```diff
@@ -13,5 +13,267 @@ kernelspec:
   name: python3
 ---
 
-# GAN (TBD)
+# Generative adversarial networks 
 
+The original purpose of GAN is to generate new data. It classically generates new images, but is applicable to wide range of domains.
+It learns the training set distribution and can generate new images that have never been seen before.
+In contrast to e.g., autoregressive models or RNNs (generating one word at a time), GANs generate the whole output all at once.
+
+```{seealso}
+GAN is proposed in 2005, the paper is [Generative Adversarial Networks](https://arxiv.org/abs/1406.2661).
+```
+
+## overview
+
+The structure of GAN is roughly as follows. It has two essential part, discriminator and generator. 
+Discriminator learns to become better at distinguishing real from generated images, and generator learns to generate better images to fool the discriminator.
+
+:::{figure-md} GAN structure
+<img src="../../images/deep-learning/GAN/structure.png" width="90%" class="bg-white mb-1">
+
+Illustration of GAN structure
+:::

```

Comment:
This problem has been discussed before. Since the images are all form the projects I mentioned in the acknowledge part, so another reference is not uncessary.